src/llm/llm_calculator.proto

```diff
@@ -27,12 +27,22 @@ message LLMCalculatorOptions {
     }
 
     enum PipelineType {
+      // Single modality (text only), text generation based on LLMPipeline
+      TEXT = 0;
+      // Multimodal (text and image), text generation based on LLMPipeline
+      VLM = 1;
+      // Single modality (text only), text generation based on ContinuousBatchingPipeline
+      TEXT_CB = 2;
+      // Multimodal (text and image), text generation based on ContinuousBatchingPipeline
+      VLM_CB = 3;
+      // Default option. If selected, pipeline type will be inferred by the serving
+      UNDEFINED_PIPELINE_TYPE = 4;
+
+      // Compatibility with current configurations, remove below in the next PR
       // Continuous batching pipeline
-      CONTINUOUS_BATCHING = 0;
+      CONTINUOUS_BATCHING = 5;

```

Comment:
isn't it duplicated?

Reply:
It has to be for now
If I remove old values, current models in CI will not load properly 
We need to merge it with old values for compatibility, regenerate models and remove them in the next PR